libs/sdk-core/src/greenlight/node_api.rs

```diff
@@ -1745,6 +1760,15 @@ impl NodeAPI for Greenlight {
         }
         Ok((hints, has_public_channel))
     }
+
+    async fn get_open_peer_channels(&self) -> NodeResult<HashMap<Vec<u8>, Channel>> {
```

Comment:
Suggestion to create a function called `get_open_peers` instead, returning a `Vec<Vec<u8>>` or `HashSet<Vec<u8>>`. The channel argument is unused. If the channel argument were used, it would have to be a vector of channels.

Reply:
Done: 1d1038149644b6c7c2718e5840d0f0c67e7f3b0a